src/main/java/controller/LadderGameController.java

```diff
@@ -0,0 +1,91 @@
+package controller;
+
+import model.*;
+import view.InputView;
+import view.OutputView;
+
+import java.util.InputMismatchException;
+
+public class LadderGameController {
+    private LadderGame ladderGame;
+    private final LadderGenerator ladderGenerator = new LadderGenerator(new LadderConnectionGenerator());
+
+    public void inputGameInfos() {
+        PlayerNames playerNames = inputPlayerNames();
+        PrizeNames prizeNames = inputPrizeNames(playerNames.size());
+        int ladderHeight = inputLadderHeight();
+
+        ladderGame = new LadderGame(ladderGenerator, playerNames, prizeNames, ladderHeight);
+    }
+
+    public void showGameStatus() {
+        OutputView.printGameStatus(ladderGame.getGameStatus());
+    }
+
+    public void showGameResult() {

```

Comment:
혹시 이쪽에 enum을 사용해보실 생각이 있으실까요? ENUM을 직접 활용한 부분이 없어서 아쉬워요
이런식으로? 사실 좀 억지로 넣는 것 같긴해도 이왕 배운거 사용해보면 좋지않을까 싶네요!
물론 여기말고 다른쪽에 활용할 부분이 있다면 거기서 enum을 사용해도 좋아요

```suggestion
public enum ResultCommand {
    ALL("all"),
    EXIT("exit"),
    SINGLE("");

    private final String command;

    ResultCommand(String command) {
        this.command = command;
    }

    public static ResultCommand from(String input) {
        if (input.equalsIgnoreCase("all")) {
            return ALL;
        }
        if (input.equalsIgnoreCase("exit")) {
            return EXIT;
        }
        return SINGLE;
    }
}
```